woocommerce/woocommerce-android
# Redesign custom amount card in order creation screen

Closes: #10238 


### Description


Custom Amount Card Redesign. 
Figma: ichqhxDjbmZZmvJaJkGhkj-fi-626_21620

**NOTE:** This cannot be merged before we add the delete custom amount button to the bottom sheet because this PR removes the delete option from the custom amount card. 

### Testing instructions


- go to order creation screen
- click on Add custom amount
- Add an amount and a name
- make sure the new card matches the design
- add more than one custom amount and make sure it looks correct
- add a custom amount with a really long name and make sure it does not break the UI

### Images/gif


| Before | After |
| ---- | ---- |
| <img src="https://github.com/woocommerce/woocommerce-android/assets/30724184/4598f995-13bc-4176-af31-43b21accc69e" width="350"/> | <img src="https://github.com/woocommerce/woocommerce-android/assets/30724184/e3f8d215-5b60-47d7-97f2-d7eba8953d79" width="350"/> |
